distributed/shuffle/tests/test_multi_file.py

```diff
@@ -0,0 +1,50 @@
+import os
```

Comment:
The implementation has drifted significantly since this file was created.  These tests no longer work.  We should fix them up.